packages/fastify-renderer/src/node/renderers/react/ReactRenderer.tsx

```diff
@@ -307,12 +311,17 @@ export class ReactRenderer implements Renderer {
           const entrypoint = id.replace(CLIENT_ENTRYPOINT_PREFIX, '/@fs/').replace(/\/hydrate\.jsx\?.+$/, '')
           const layout = url.searchParams.get('layout')!
           const base = url.searchParams.get('base')!
+          const route = url.searchParams.get('route')!
```

Comment:
Could we maybe name this `imperativeRoute` or something like that just to make it really clear what is going on? `route` makes me think it might be used all the time on purpose